[ADD] accounting/localization: l10n_pe functional documentation

ren-odoo opened the pull request:
Functional documentation for the Peruvian Localization: 

- Intro
- Configuration
- Tesiting and usage

ren-odoo commented:
Hi @jcs-odoo 

Can  you help to take a look at this PR please, we're planning to merge l10n_pe_edi tomorrow (monday 25th), so the sooner we can published the Functional documentation the better. 
 
Note: I got a warining in line 128 due to the table format, but I followed the formating instructions from RST cheat sheet, let me know what change is needed. 

Thank you in advance.

jcs-odoo commented on accounting/fiscal_localizations/localizations/peru.rst:
```diff
@@ -0,0 +1,535 @@
+To finish the workflow please follow the instructions on 
+`this section <https://www.odoo.com/documentation/user/14.0/accounting/receivables/customer_invoices/credit_notes.html>`_.
+
+.. note::
+   The EDI workflow for the SUNAT in the same way as the invoices. 
```
I think a part of the sentence is missing.

jcs-odoo commented on accounting/fiscal_localizations/localizations/peru.rst:
```diff
@@ -0,0 +1,535 @@
+
+Debit Notes
+------------
+As part of the Preuvian localization, besides creating credit notes from an existing document 
```
```suggestion
As part of the Peruvian localization, besides creating credit notes from an existing document 
```

jcs-odoo commented:
I forgot to ask:
is the localization only available on odoo 14?
Otherwise, would this doc be also valid for older versions?

ren-odoo commented:
> I forgot to ask:
> is the localization only available on odoo 14?
> Otherwise, would this doc be also valid for older versions?

Yes, this module will be released, only on V14.

ren-odoo commented:
> * accounting/fiscal_localizations/overview/localizations_list.rst

Hi @jcs-odoo  thank you for the feedback 

I have addressed all the comments and commited your suggestions as well. 

Regarding the changes on accounting/fiscal_localizations/overview/localizations_list.rst, the Colombian ones apply on V13 as well so I remove them from here to latter create a PR just for that on V13. 

Let me know if there is anything else. 

Thank you.

jcs-odoo reviewed (commented):
just a few comments :)

ren-odoo commented on accounting/fiscal_localizations/localizations/peru.rst:
```diff
@@ -0,0 +1,583 @@
+Signature Provider
+~~~~~~~~~~~~~~~~~~
+
+As part of  the requirements for Electronic Invoice is Peru, your company needs to select a
```
Fixed in last commit.

ren-odoo commented on accounting/fiscal_localizations/localizations/peru.rst:
```diff
@@ -0,0 +1,583 @@
+.. important::
+   If you have a validation error and the document needs to be sent one more time, an additional 
+   credit will be charged. Therefore, it is paramount that you verify all information is correct 
+   before sending your document to the OSE.
```
Fixed in last commit.

ren-odoo commented on accounting/fiscal_localizations/localizations/peru.rst:
```diff
@@ -0,0 +1,583 @@
+
+- **l10n_pe_edi**: includes all technical and functional requirements to generate and validate 
+  Electronic Invoice, based on the SUNAT specification to create and process valid electronic 
+  documents, for more technical detail you can access to   
```
Fixed in last commit.
